src/main/java/gov/hhs/cms/bluebutton/fhirstress/utils/RifEntry.java

```diff
@@ -0,0 +1,66 @@
+package gov.hhs.cms.bluebutton.fhirstress.utils;
+
+import com.opencsv.bean.CsvBindByName;
+
+public class RifEntry 
```

Comment:
You probably ought to reuse the RIF parsing code in `/bluebutton-data-model-rif/gov.hhs.cms.bluebutton.data.model.rif.BeneficiaryParser`.

Reply:
Done, but used RifRecordEvents and RifFilesProcessor which uses BeneficiaryParser.